rma_account/views/rma_order_line_view.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record id="view_rma_line_supplier_form" model="ir.ui.view">
+        <field name="name">rma.order.line.supplier.form</field>
+        <field name="model">rma.order.line</field>
+        <field name="inherit_id" ref="rma.view_rma_line_supplier_form" />
+        <field name="arch" type="xml">
+            <button name="action_view_out_shipments" position="after">
+                <button
+                    type="object"
+                    name="action_view_invoice"
+                    class="oe_stat_button"
+                    icon="fa-pencil-square-o"
+                    string="Origin Inv"
+                >
+                </button>
+                <button
+                    type="object"
+                    name="action_view_refunds"
+                    class="oe_stat_button"
+                    icon="fa-pencil-square-o"
+                    groups="account.group_account_user"
+                >
+                        <field name="refund_count" widget="statinfo" string="Refunds" />
+                    </button>
+            </button>
+            <group name="main_info" position="inside">
+                <field
+                    name="account_move_line_id"
+                    options="{'no_create': True}"
+                    context="{'rma': True}"
+                    domain="['|',
```

Comment:
```suggestion
                    domain="[('move_id.type', 'not in', ('entry','out_invoice','out_refund')), '|',
```

Reply:
yes but an array, not a tuple after not in